test/core/end2end/fuzzers/api_fuzzer.proto

```diff
@@ -95,12 +85,12 @@ message Metadatum {
 
 message CreateChannel {
     string target = 1;
-    repeated ChannelArg channel_args = 2;
+    grpc.testing.FuzzingChannelArgs channel_args = 2;
```

Comment:
To avoid breaking existing corpora entries, we could leave the existing field in place but add the new field as well, and then have the API fuzzer code ignore the old field if the new field is set.

This may or may not be worth doing, but I figured I'd offer the suggestion.

Reply:
Given that the api fuzzer corpus only has 2 items with channel_args specified, I'd be more inclined to change those two to match the new format than to keep the tech debt of both ChannelArgs proto specifications. And I also trust Craig that the entries would be regenerated pretty quickly if regressions occurred.

We could aim for backwards compatibility with the text-protos that we keep in the corpus, but I think managing the collection would be a bit messier - each fuzzer would have a collection of `FuzzingChannelArg`, instead of a single `FuzzingChannelArgs`, and we might need a templated creation method.